### ISSUE TITLE
(Urgent) 404 on PDF

### ISSUE THREAD

**Issue by ghost** [Anon-Planet/thgtoa#110]
https://anonymousplanet-ng.org/export/guide.pdf > 404 errror

**Comment by pterocles** [comment]
Not sure what's going on. 

Here are all files being archived during the build process: everything except guide.pdf and export/guide.pdf.

```
./minisign.pub
./links.html
./links.md
./robots.txt
./KEY_ROTATION.md.asc
./make.sh
./legacy.md
./LICENSE.md
./verify.md
./KEY_ROTATION.html
./index.html
./KEY_ROTATION.md
./LICENSE.html
./sha256sum.txt
./briar.html
./sitemap.xml
./CHANGELOG.md
./export/
./export/CONTRIBUTING.odt.asc
./export/KEY_ROTATION.pdf.minisig
./export/links.html
./export/CODE_OF_CONDUCT.pdf.asc
./export/chatrooms-rules.html.asc
./export/CONTRIBUTING.pdf.asc
./export/CONTRIBUTING.html.asc
./export/links.odt
./export/CHANGELOG.odt
./export/moderncrypto-rules.pdf
./export/guide.odt.asc
./export/briar.odt.minisig
./export/KEY_ROTATION.odt
./export/verify.pdf.minisig
./export/CHANGELOG.pdf
./export/legacy.odt.asc
./export/KEY_ROTATION.html
./export/moderncrypto-rules.odt
./export/donations.pdf
./export/links.pdf.asc
./export/CONTRIBUTING.odt
./export/about.odt
./export/links.html.minisig
./export/LICENSE.pdf.asc
./export/about.odt.asc
./export/LICENSE.html
./export/moderncrypto-rules.html.minisig
./export/sha256sum.txt
./export/donations.html.minisig
./export/briar.html
./export/LICENSE.pdf.minisig
./export/donations.pdf.asc
./export/donations.html.asc
./export/CODE_OF_CONDUCT.odt.asc
./export/chatrooms-rules.pdf.minisig
./export/briar.pdf.asc
./export/README.html.asc
./export/LICENSE.odt.minisig
./export/about.html.asc
./export/chatrooms-rules.odt.minisig
./export/guide.odt
./export/README.pdf
./export/CHANGELOG.pdf.minisig
./export/CODE_OF_CONDUCT.html
./export/links.html.asc
./export/KEY_ROTATION.html.asc
./export/links.odt.minisig
./export/KEY_ROTATION.pdf
./export/LICENSE.odt.asc
./export/sha256sum.txt.asc
./export/legacy.pdf.asc
./export/legacy.html
./export/about.pdf.minisig
./export/KEY_ROTATION.html.minisig
./export/legacy.pdf
./export/README.html.minisig
./export/chatrooms-rules.odt
./export/briar.pdf
./export/moderncrypto-rules.odt.asc
./export/CODE_OF_CONDUCT.html.minisig
./export/legacy.html.asc
./export/CODE_OF_CONDUCT.html.asc
./export/donations.pdf.minisig
./export/CODE_OF_CONDUCT.odt.minisig
./export/briar.html.minisig
./export/chatrooms-rules.html.minisig
./export/LICENSE.pdf
./export/b2sum.txt.asc
./export/chatrooms-rules.pdf
./export/legacy.odt.minisig
./export/guide.html.asc
./export/LICENSE.odt
./export/CONTRIBUTING.pdf.minisig
./export/LICENSE.html.asc
./export/verify.odt
./export/chatrooms-rules.odt.asc
./export/moderncrypto-rules.html.asc
./export/CHANGELOG.odt.minisig
./export/guide.html
./export/README.html
./export/moderncrypto-rules.html
./export/about.pdf.asc
./export/CHANGELOG.odt.asc
./export/legacy.pdf.minisig
./export/about.pdf
./export/verify.html.minisig
./export/verify.odt.asc
./export/about.html
./export/chatrooms-rules.html
./export/README.pdf.asc
./export/verify.odt.minisig
./export/CODE_OF_CONDUCT.odt
./export/legacy.odt
./export/donations.odt.minisig
./export/verify.pdf.asc
./export/CONTRIBUTING.html.minisig
./export/KEY_ROTATION.pdf.asc
./export/README.odt.minisig
./export/chatrooms-rules.pdf.asc
./export/about.html.minisig
./export/CONTRIBUTING.pdf
./export/KEY_ROTATION.odt.minisig
./export/links.pdf.minisig
./export/verify.pdf
./export/b2sum.txt
./export/briar.html.asc
./export/KEY_ROTATION.odt.asc
./export/b2sum.txt.minisig
./export/briar.pdf.minisig
./export/briar.odt
./export/guide.odt.minisig
./export/moderncrypto-rules.odt.minisig
./export/CHANGELOG.html.minisig
./export/README.pdf.minisig
./export/LICENSE.html.minisig
./export/README.odt
./export/about.odt.minisig
./export/briar.odt.asc
./export/moderncrypto-rules.pdf.asc
./export/verify.html.asc
./export/CONTRIBUTING.odt.minisig
./export/README.odt.asc
./export/donations.odt
./export/donations.html
./export/CODE_OF_CONDUCT.pdf.minisig
./export/CODE_OF_CONDUCT.pdf
./export/CHANGELOG.pdf.asc
./export/guide.html.minisig
./export/links.pdf
./export/CHANGELOG.html
./export/donations.odt.asc
./export/CHANGELOG.html.asc
./export/legacy.html.minisig
./export/moderncrypto-rules.pdf.minisig
./export/sha256sum.txt.minisig
./export/links.odt.asc
./export/verify.html
./export/CONTRIBUTING.html
./CONTRIBUTING.md
./sha256sum.txt.asc
./legacy.html
./clean.sh
./b2sum.txt.asc
./guide.html
./moderncrypto-rules.html
./guide.md
./assets/
./assets/css/
./assets/css/style.css
./assets/css/light_style.css
./assets/images/
./assets/images/bullet_light.png
./assets/images/blacktocat.png
./assets/images/bkg.png
./assets/images/bullet.png
./about.html
./chatrooms-rules.md
./chatrooms-rules.html
./about.md
./KEY_ROTATION.md.minisig
./media/
./media/bitcoin-segwit.png
./media/image21.jpeg
./media/image42.jpeg
./media/image16.jpeg
./media/image8.jpeg
./media/image29.jpeg
./media/image32.jpeg
./media/image47.jpeg
./media/image4.jpeg
./media/image24.jpeg
./media/image11.jpeg
./media/image48.jpeg
./media/after.gif
./CNAME
./Makefile
./CODE_OF_CONDUCT.md
./42FF35DB9DE7C088AB0FD4A70C216A52F6DF4920.asc
./b2sum.txt
./b2sum.txt.minisig
./README.md
./donations.html
./briar.md
./moderncrypto-rules.md
./CHANGELOG.html
./donations.md
./sha256sum.txt.minisig
./verify.html
```

**Comment by pterocles** [comment]
The runner doesn't show any errors.

**Comment by pterocles** [comment]
Unfortunately, it's building `github-pages` without returning useful errors, and this is done via the `_config.yml` for the workflow which I can't access. It's something the admin will have to look into themselves. I bet you it's a Cyrillic character issue. Did we add any to the guide through that last batch of PRs?

**Comment by NobodySpecial256** [comment]
The PDF isn't built by GitHub, it's built manually using [make.sh](https://github.com/Anon-Planet/thgtoa/blob/master/make.sh)

**Comment by NobodySpecial256** [comment]
```
Generating PDF...
[WARNING] Note with key '363' defined at line 12994 column 1 but not used.
[WARNING] Note with key '459' defined at line 13186 column 1 but not used.
[WARNING] Note with key '460' defined at line 13188 column 1 but not used.
[WARNING] Note with key '465' defined at line 13198 column 1 but not used.
[WARNING] Note with key '483' defined at line 13234 column 1 but not used.
Error producing PDF.
tex error       > tex error on line 19957 in file /tmp/tex2pdf.-ba7d5f8ff46c02ea/input.tex: ! Undefined control sequence

Generating ODT...
```

That's the error output I get when I build the PDF

**Comment by pterocles** [comment]
> ```
> Generating PDF...
> [WARNING] Note with key '363' defined at line 12994 column 1 but not used.
> [WARNING] Note with key '459' defined at line 13186 column 1 but not used.
> [WARNING] Note with key '460' defined at line 13188 column 1 but not used.
> [WARNING] Note with key '465' defined at line 13198 column 1 but not used.
> [WARNING] Note with key '483' defined at line 13234 column 1 but not used.
> Error producing PDF.
> tex error       > tex error on line 19957 in file /tmp/tex2pdf.-ba7d5f8ff46c02ea/input.tex: ! Undefined control sequence
> 
> Generating ODT...
> ```
> 
> That's the error output I get when I build the PDF

Fixed and building in #111 https://github.com/Anon-Planet/thgtoa/pull/111/commits/b8939c9e21a91fe299be7dedcd4bb552ea23f966.
